pkg/services/object/acl/v2/service.go

```diff
@@ -82,9 +82,9 @@ func New(opts ...Option) Service {
 	}
 }
 
-// ProcessGetRequest resolves RequestInfo from the request to check it using
+// GetRequestToInfo resolves RequestInfo from the request to check it using
 // [ACLChecker].
-func (b Service) ProcessGetRequest(request *protoobject.GetRequest) (RequestInfo, error) {
+func (b Service) GetRequestToInfo(request *protoobject.GetRequest) (RequestInfo, error) {
```

Comment:
i do not feel it is now better. "there is no processing there" but it asserts smth, parses, tries to understand what exact field is about the request author, etc

Reply:
Still just a conversion to me. Processing is performing the essence of some operation, this one simply checks/collects some input data. No decisions made except for protocol errors.